pages/common/git-sed.md

```diff
@@ -0,0 +1,22 @@
+# git sync
+
+> Run grep as directed but replace the given files with the pattern.
```

Comment:
This description says nothing to me.

Reply:
I have updated the description on 226a057 please let me know if that is better.